packages/playwright/test/RunPartial/UseLegacyTests.cs

```diff
@@ -0,0 +1,78 @@
+using System.Threading.Tasks;
+using System.Linq;
+using NUnit.Framework;
+
+namespace Deque.AxeCore.Playwright.Test.RunPartial
+{
+
+    [Category("RP")]
```

Comment:
Is this intended to represent "RunPartial"? Is it not redundant with the `[Category("RunPartial")]` on `TestBase`?

Reply:
Whoops. Added that so I could quickly run just those tests through `--filter Category=RP`. Forgot to remove